app/views/friends/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= form_with(model: friend, local: true) do |form| %>
+  <% if friend.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(friend.errors.count, "error") %> prohibited this friend from being saved:</h2>
+
+      <ul>
+        <% friend.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
You should not add logic in the view. If you need to use this logic, encapsulate in View helpers `(app/helpers/*_helper.rb)`.

Reply:
I honestly don't understand your suggestions, we used the scaffold to create this for us, this was done by the scaffold, what you are asking us to do goes against what we are doing and using.